@noun-auction/data/activeNounishAuction.ts

```diff
@@ -0,0 +1,40 @@
+import { returnMarketProps } from '@noun-auction/constants'
+import { NounishMarketTypes } from '@noun-auction/typings'
+
+export function activeNounishAuction(marketType?: string) {
+  const marketProps = returnMarketProps(marketType as NounishMarketTypes)
```

Comment:
`marketType` probably can be moved to function params

Reply:
Good shout. Modified this in all instances where it's referenced.